### PR TITLE
Release/3.8.0

### PR DESCRIPTION
**Pull request by mehmet-yoti**

### CODE REVIEW

**Comment by mehmet-yoti** [comment]
*Added below functions
- Support for non latin documents at session creation
- Support for non latin documents at session creation
- Support for non latin documents when fetching supported documents
- Support for expired documents at session creation
- Static liveness check

File: _examples/idv/models.sessionspec.go [hunk 1]
```diff
@@ -150,6 +155,55 @@ func buildSessionSpec() (sessionSpec *create.SessionSpecification, err error) {
 		WithRequiredDocument(supplementaryDoc).
 		Build()
 
+	fmt.Println("%v", sessionSpec)
```
Comment (fofiuiancu) [inline _examples/idv/models.sessionspec.go:158]: leftover debug code?

File: docscan/client.go [hunk 2]
```diff
@@ -242,13 +242,43 @@ func (c *Client) DeleteMediaContent(sessionID, mediaID string) error {
 	return nil
 }
 
+// DeleteSession deletes a previously created Yoti Doc Scan (IDV) session and all of its related resources
```
Comment (fofiuiancu) [inline docscan/client.go:245]: mismatch between the comment and the function name

**Review by fofiuiancu — CHANGES_REQUESTED** [review]
- Sonar Scan failures
- it seems that the branch needs to be rebased on development so 3.7.0 changes do not appear here
- all the feature branches were merged without being reviewed. I'd suggest deleting this branch and re-open each feature branch and only after all have been merged (through a PR) in development to re-create this release/3.8.0 branch 
- both feature PRs and Release PRs descriptions will have to comply with https://keepachangelog.com/en/1.0.0/
- it seems that changes were added after QA signed off?
- not needed files were added (.DS_Store files and the idv executable)

**Pull request closed without merging**